ktor-client/ktor-client-plugins/ktor-client-resources/common/test/ResourcesTest.kt

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+import io.ktor.client.engine.mock.*
+import io.ktor.client.plugins.resources.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.client.tests.utils.*
+import io.ktor.http.*
+import io.ktor.resources.*
+import kotlinx.serialization.*
+import kotlin.test.*
+
+class ResourcesTest {
+
+    @Serializable
+    @Resource("path/{id}/{method}")
```

Comment:
btw could we use `Resource` with `gson`?

Reply:
what do you mean? `gson` is not multiplatform and does not provide pluggable formats